docs/02-quick-start/01-installation/02-kubernetes.md

```diff
@@ -1,7 +1,106 @@
+  limitations under the License.
+-->
 
 # Try Ozone With Kubernetes
 
-**TODO:** File a subtask under [HDDS-9856](https://issues.apache.org/jira/browse/HDDS-9856) and complete this page or section.
+## Minikube
+
+**Requirements:**
+
+* Working minikube setup
+* kubectl
+
+`kubernetes/examples` folder of the Ozone distribution contains Kubernetes deployment resource files for multiple use cases. By default the Kubernetes resource files are configured to use *apache/ozone* image from the Docker Hub.
+
+To deploy it to minikube, use the minikube configuration set:
+
+``` bash
+cd kubernetes/examples/minikube
+kubectl apply -f .
+```
+
+And you can check the results with
+
+``` bash
+kubectl get pod
+```
+
+Note: the `kubernetes/examples/minikube` resource set is optimized for minikube usage:
+
+* You can have multiple Datanodes even if you have only one host (in a real production cluster usually you need one Datanode per physical host)
+* The services are published with node port
+
+### Access the services
+
+Now you can access any of the services. For each web endpoint an additional NodeType service is defined in the minikube k8s resource set. NodeType services are available via a generated port of any of the host nodes:
+
+``` bash
+kubectl get svc
+NAME         TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE
+datanode     ClusterIP   None            <none>        <none>           27s
+kubernetes   ClusterIP   10.96.0.1       <none>        443/TCP          118m
+om           ClusterIP   None            <none>        9874/TCP         27s
+om-public    NodePort    10.108.48.148   <none>        9874:32649/TCP   27s
+s3g          ClusterIP   None            <none>        9878/TCP         27s
+s3g-public   NodePort    10.97.133.137   <none>        9878:31880/TCP   27s
+scm          ClusterIP   None            <none>        9876/TCP         27s
+scm-public   NodePort    10.105.231.28   <none>        9876:32171/TCP   27s
+```
+
+Minikube contains a convenience command to access any of the NodePort services:
+
+``` bash
+minikube service s3g-public
+Opening kubernetes service default/s3g-public in default browser...
+```
+
+## Hosted Kubernetes Cluster
+
+**Requirements:**
+
+* Working Kubernetes cluster (LoadBalancer, PersistentVolume are not required)
+* kubectl
+
+As *apache/ozone* Docker images are available from Docker Hub, the deployment process is very similar to Minikube deployment. The only big difference is that we have dedicated set of k8s files for hosted clusters (for example we can use one Datanode per host)
+
+### Deploy to Kubernetes
+
+`kubernetes/examples` folder of the Ozone distribution contains Kubernetes deployment resource files for multiple use cases.
+
+To deploy to a hosted cluster use the Ozone subdirectory:
+
+``` bash
+cd kubernetes/examples/ozone
+kubectl apply -f .
+```
+
+And you can check the results with
+
+``` bash
+kubectl get pod
+Access the services
+```
+
```

Comment:
This is also supposed to be a heading:

```
### Access the services
```

Reply:
If I add the heading, it would result in the MD024/no-duplicate-heading error at /root/project/ozone-site/docs/02-quick-start/01-installation/02-kubernetes.md:100 under '### Access the services'. Should I simply drop the heading?

Reply:
Hmm, good point.

Possible solutions:
- split the page, as in the existing website
- come up with unique headings
- remove these subsection headings
- is there some flag to allow duplicate headings?

@errose28 what's your take?

Reply:
Ah this was a miss when I set up the CI. We only want the duplicate header check for siblings, which is not the case here. See https://github.com/DavidAnson/markdownlint/blob/v0.35.0/doc/md024.md and please set `siblings_only=true` in `markdownlint.yaml` as part of this PR to fix this issue. Then these headings should be allowed.